perun-core/src/main/java/cz/metacentrum/perun/core/impl/modules/attributes/urn_perun_group_attribute_def_def_synchronizationEnabled.java

```diff
@@ -33,6 +39,16 @@ public void checkAttributeValue(PerunSessionImpl sess, Group group, Attribute at
 		if(!attrValue.equals("true") && !attrValue.equals("false")) {
 			throw new WrongAttributeValueException(attribute, group, "If attribute is not null, only string 'true' or 'false' is correct format.");
 		}
+		try {
+			if (attrValue.equals("true")
+			        && (sess.getPerunBl().getAttributesManagerBl().getAttribute(sess, group, GROUPSYNCHROINTERVAL_ATTRNAME).getValue() == null
+			        || sess.getPerunBl().getAttributesManagerBl().getAttribute(sess, group, GROUPMEMBERSQUERY_ATTRNAME).getValue() == null
+			        || sess.getPerunBl().getAttributesManagerBl().getAttribute(sess, group, GROUPEXTSOURCE_ATTRNAME).getValue() == null)) {
+				throw new WrongReferenceAttributeValueException("At least one of required attributes " +  GROUPSYNCHROINTERVAL_ATTRNAME + ", " + GROUPMEMBERSQUERY_ATTRNAME + " or " + GROUPEXTSOURCE_ATTRNAME + " is not set.");
```

Comment:
This exception can hold actual attributes, which is most convenient in GUI, where we state which attributes are wrong and what is their value. Can you please split this throwing into 3 with proper attributes passed to the exception. Thank you.